addons/manager.go

```diff
@@ -223,21 +230,21 @@ func runSpokeManager(ctx context.Context, options AddonAgentOptions) {
 	})
 
 	if err != nil {
-		klog.Error(err, "unable to start manager")
+		logger.Error("Failed to start manager", "error", err)
 		os.Exit(1)
 	}
 
 	spokeKubeConfig := mgr.GetConfig()
 	spokeKubeClient, err := kubernetes.NewForConfig(spokeKubeConfig)
 	if err != nil {
-		klog.Error(err, "unable to get spoke kube client")
+		logger.Error("Failed to get spoke kube client", "error", err)
 		os.Exit(1)
 	}
 
 	currentNamespace := os.Getenv("POD_NAMESPACE")
 
 	if err = mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
-		klog.Infof("Waiting for MaintenanceMode CRD to be created. MaintenanceMode controller is not running yet.")
+		logger.Info("Waiting for MaintenanceMode CRD to be established")
```

Comment:
I think the existing message is more clearer about the state of the system.

Reply:
changed